zombies.js

```diff
@@ -79,7 +88,123 @@
  * @property {method} getMaxHealth         Returns private variable `maxHealth`.
  */
 
+function Player(name, health,strength, speed){
+  this.name = name;
+  this.health = health;
+  this.strength = strength;
+  this.speed = speed;
+  this.isAlive = true;
+  this._pack = [];
+  this._maxHealth = health;
+  this.equipped = false;
+
+  this.getPack = function(){
+    return this._pack;
+  }
+
+  this.getMaxHealth = function(){
+    return this._maxHealth;
+  }
+
+  this.takeItem = function(item){
```

Comment:
From this method down, they should be placed on the prototype. The way they currently are, they are on the instance object only and if you extended player, these methods wouldn't follow.